### DISCUSSION TITLE
About mania touching input system

### DISCUSSION THREAD

**Discussion by DrrrrrrZM** [ppy/osu#28893] [Ideas]
The new touching input system (in the picture) is not suitable for everyone. Please create a mania touching input system option in the next version.(Just remain the old or use new input system)
![Screenshot_2024-07-17-22-09-03-405_com android browser-edit](https://github.com/user-attachments/assets/53c99c53-e4fb-4b80-a10f-49a707b733cb)

**Comment by Zzy-114514** [comment]
Yes, especially on mobile devices, where it even feels weird.

**Comment by FelynnRaynard** [comment]
I agree. It makes it too difficult for me. Now, I can't play the 3-star maps I used to be able to because it's made hitting notes harder than before
